purchase_line_product_image/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
+
+Purchase Order Line Product Image for purchases
+==========================================================
+
+Show Product Image at Purchase Order Line.
+
+We can see image from product, at purchase order line.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/10.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/purchase-workflow/issues>`_.
```

Comment:
This is the old verbiage. Please update to align with [the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#bug-tracker)